SpringCollab2020Module.cs

```diff
@@ -14,9 +15,21 @@ public class SpringCollab2020Module : EverestModule {
         }
 
         public override void Load() {
+            Everest.Events.Level.OnExit += onLevelExit;
         }
 
         public override void Unload() {
-        }
+            Everest.Events.Level.OnExit -= onLevelExit;
+        }
+
+        private void onLevelExit(Level level, LevelExit exit, LevelExit.Mode mode, Session session, HiresSnow snow) {

```

Comment:
This isn't a complaint with your hook, but something I've just realized in general:

Maybe entity-specific hooks should be contained in the entity class, with the module's Load and Unload methods calling static Load and Unload methods in those classes.

(I'll crosspost this onto the collabcord.)

Reply:
Yeah, that's a good idea, it will prevent the module class from becoming a grab bag of hooks for everything 😅 that will spare us a refactor later